gulpfile.js

```diff
@@ -46,12 +46,24 @@ gulp.task('run-tests', ['lint', 'test-coverage']);
 gulp.task('build', ['build-bundle-prod']);
 
 gulp.task('clean', function () {
-  return gulp.src(['build'], {
-      read: false
-    })
-    .pipe(clean());
+  clean('');
+});
+
+gulp.task('clean-dev', function () {
+  clean('/dev');
 });
 
+gulp.task('clean-dist', function () {
+  clean('/dist');
+});
+
+function clean(type) {
```

Comment:
I think this needs to be renamed to something else. It clobbers the import: `var clean = require('gulp-clean');`

As written, I think it's actually unused. You can tell by doing the following:

```
gulp build
gulp clean-dist
```

expected: The `build/` directory should either be empty, or not exist.
observed: There's still a `build/dist` directory with stuff in it